android/build.gradle

```diff
@@ -1,22 +1,27 @@
 buildscript {
     repositories {
+        google()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.android.tools.build:gradle:3.4.2'
```

Comment:
It looks like `react-native-permissions` [might be a good example to follow for this file](https://github.com/react-native-community/react-native-permissions/blob/373fbd8f5c7085cf87bfe17320c804d1c744ccf8/android/build.gradle).